pulp_container/app/downloaders.py

```diff
@@ -24,13 +27,14 @@ class RegistryAuthHttpDownloader(HttpDownloader):
     """
 
     registry_auth = {"bearer": None, "basic": None}
-    token_lock = asyncio.Lock()
 
     def __init__(self, *args, **kwargs):
         """
         Initialize the downloader.
         """
         self.remote = kwargs.pop("remote")
+        self.token_lock = asyncio.Lock()
```

Comment:
This change was made because of this: https://github.com/pulp/pulp_container/actions/runs/6686219325/job/18165335541#step:12:2059. Not sure what is the performance impact.

Reply:
What is the lock even good for when you have it per instance instead of per class. I don't think this is how it was intended.